VatSpace/main.lua

```diff
@@ -36,7 +36,14 @@ end
 
 local function restartGameForNextLevel()
     if gameTimer then
-        stopTimer()        
+        stopTimer() 
+        asteroids.destroy()
+        asteroids.create()
```

Comment:
Hmm, what do you think about create method recreate() in asteroids to encapsulate these two functions?